### DISCUSSION TITLE
We updated our RSA SSH host key

### DISCUSSION THREAD

**Discussion by Akash1134** [community/community#50889] [Announcements]
At approximately 05:00 UTC on March 24, out of an abundance of caution, we replaced our RSA SSH host key used to secure Git operations for GitHub.com. We did this to protect our users from any chance of an adversary impersonating GitHub or eavesdropping on their Git operations over SSH.

You may receive a warning message which is expected for users that had our previous key verified. **Please refer to [our post](https://github.blog/2023-03-23-we-updated-our-rsa-ssh-host-key/) on the GitHub Blog for more information and remediation steps.**

If you have any further questions and feedback to share, please post them in this [discussion](https://github.com/orgs/community/discussions/50878).

**Comment by novium2** [comment]
Retiring this post as it is over 6 months old. If you have any questions, consider starting a [new post](https://github.com/orgs/community/discussions/new/choose).
